pkg/queue/breaker_test.go

```diff
@@ -86,7 +86,7 @@ func TestBreakerReserveOverload(t *testing.T) {
 }
 
 func TestBreakerOverloadMixed(t *testing.T) {
-	// This tests when reservation and maybe are intermised.
+	// This testCases when reservation and maybe are intermised.
```

Comment:
this change seems superfluous.

Reply:
Yeah I am not done yet this is refactoring relic.